packages/text-block/src/TextBlock.tsx

```diff
@@ -1,39 +1,51 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { useBlockSettings, useEditorState } from '@frontify/app-bridge';
-import { RichTextEditor, defaultPlugins, defaultPluginsWithColumns } from '@frontify/fondue';
+import { RichTextEditor } from '@frontify/fondue';
 import '@frontify/fondue-tokens/styles';
 import { BlockProps } from '@frontify/guideline-blocks-settings';
-import { useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
-import { FC } from 'react';
+import { RichTextEditorToHtml, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
 import 'tailwindcss/tailwind.css';
 import { PLACEHOLDER } from './settings';
 import { Settings, spacingValues } from './types';
+import { getPlugins } from './getPlugins';
+import { ReactElement } from 'react';
 
-export const TextBlock: FC<BlockProps> = ({ appBridge }) => {
+export const TextBlock = ({ appBridge }: BlockProps): ReactElement => {
     const isEditing = useEditorState(appBridge);
     const [blockSettings, setBlockSettings] = useBlockSettings<Settings>(appBridge);
     const { designTokens } = useGuidelineDesignTokens();
-
-    const onTextChange = (value: string) => value !== blockSettings.content && setBlockSettings({ content: value });
     const gap = blockSettings.isColumnGutterCustom
         ? blockSettings.columnGutterCustom
         : spacingValues[blockSettings.columnGutterSimple];
 
+    const onTextChange = (value: string) => value !== blockSettings.content && setBlockSettings({ content: value });
+
     return (
-        <div data-test-id="text-block">
-            <RichTextEditor
-                designTokens={designTokens ?? undefined}
-                key={'text-block-editor'}
-                value={blockSettings.content}
-                layout={{ gap, columns: blockSettings.columnNumber }}
-                border={false}
-                plugins={blockSettings.columnNumber > 1 ? defaultPluginsWithColumns : defaultPlugins}
-                placeholder={isEditing ? PLACEHOLDER : undefined}
-                readonly={!isEditing}
-                onTextChange={onTextChange}
-                onBlur={onTextChange}
-            />
-        </div>
+        <>
+            {!isEditing ? (
+                <RichTextEditorToHtml
+                    columnGap={gap}
+                    columns={blockSettings.columnNumber}
+                    content={blockSettings.content ?? ''}
+                    designTokens={designTokens}
+                />
+            ) : (
+                <RichTextEditor
+                    data-test-id="rich-text-editor"
+                    id={appBridge.getBlockId().toString()}
+                    designTokens={designTokens ?? undefined}
```

Comment:
```suggestion
                    designTokens={designTokens}
```

Reply:
atm designTokens still returns null. But as soon as we have always the default values, we have to remove it. Agreed.